src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -119,6 +119,7 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
               match e with
               | GetError.OneWayToSource -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Binding {BindingName} is read-only", nameChain, binder.Name)
               | GetError.SubModelSelectedItem d -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Failed to find an element of the SubModelSeq binding {SubModelSeqBindingName} with ID {ID} in the getter for the binding {BindingName}", d.NameChain, d.SubModelSeqBindingName, d.Id, binder.Name)
+              | GetError.ToNullError (ValueOption.ToNullError.ValueCannotBeNull nonNullTypeName) -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Binding {BindingName} is null, but type {Type} is non-nullable", nameChain, binder.Name, nonNullTypeName)
```

Comment:
From https://github.com/elmish/Elmish.WPF/pull/470#discussion_r963188796:

> Can you add the non-nullable type to this error message?

-- @TysonMN

Reply:
This is done